### PR TITLE
fix cluster and vcenter order

### PR DESCRIPTION
**Pull request by zhoumeina**
in create vch wizard. In select "compute resource" view. 
The vcenter and clusters and hosts are not in correct order. That is because we get the data directly from vsphere sdk without sort them. 
fix:
In the ui view, add sort function to all those list to keep align with vcenter tree view.

### CODE REVIEW

File: h5c/vic/src/vic-webapp/src/app/create-vch-wizard/create-vch-wizard.service.ts [hunk 1]
```diff
@@ -196,11 +196,14 @@ export class CreateVchWizardService {
      * @returns {Observable<ComputeResource[]>}
      */
     getResourcesCompleteInfo(basicInfoList: ResourceBasicInfo[]): Observable<ComputeResource[]> {
-      const infoListObs: Observable<ComputeResource>[] = basicInfoList
+      if (basicInfoList) {
+        const infoListObs: Observable<ComputeResource>[] = basicInfoList
```
Comment (wjun) [inline h5c/vic/src/vic-webapp/src/app/create-vch-wizard/create-vch-wizard.service.ts]: Format is required.

Comment (zhoumeina) [reply h5c/vic/src/vic-webapp/src/app/create-vch-wizard/create-vch-wizard.service.ts]: done.

**Comment by renmaosheng** [comment]
do we need to change the test as well? thanks.

**Comment by zhoumeina** [comment]
@remaosheng, we have some tests releated with change. I need to check with wanngjun with it.

**Pull request merged**